scilpy/tracking/tracker.py

```diff
@@ -535,7 +561,7 @@ def track(self):
         # Convert theta to cos(theta)
         max_cos_theta = np.cos(np.deg2rad(self.theta))
 
-        cl_kernel = CLKernel('tracker', 'tracking', 'local_tracking.cl')
```

Comment:
Not sure if this change is supposed to be there.

Reply:
Almost sure it shouldn't be there! there seems to have been a problem when merging with the master branch.